src/app/pages/Kafka/ConnectedMetrics.tsx

```diff
@@ -7,41 +7,33 @@ import { fetchKafkaInstanceMetrics, fetchKafkaTopisFromAdmin, fetchMetricsKpi, f
 type ConnectedMetricsProps = {
   kafkaId: string;
   kafkaAdminUrl: string;
-  instanceType: "standard" | "trial"
 };
 
-export const ConnectedMetrics: VoidFunctionComponent<ConnectedMetricsProps> = ({ kafkaId, kafkaAdminUrl, instanceType }) => {
+export const ConnectedMetrics: VoidFunctionComponent<ConnectedMetricsProps> = ({ kafkaId, kafkaAdminUrl }) => {
   const auth = useAuth();
   const history = useHistory();
   const config = useConfig();
   const { getBasename } = useBasename() || {};
   const basename = getBasename && getBasename();
 
+  const [isAlertClosed, setIsAlertClosed] = useState<boolean>(false);
```

Comment:
I think we don't need to lift this state here. We should manage this state  (isAlertClosed) in ui-components. It's for closing alert , there is no use in parent component.
@riccardo-forina wdyt ?

Reply:
Agree

Reply:
That is, unless this needs to never open again for the user by storing somehow - localStorage - that the user closed it. @jgiardino is this what you want here?

Reply:
Ideally this would be as persistent as possible. I'm not sure what our options are with local storage and if this would be dismissed per Kafka instance or for the entire service. I don't see this as a blocker, though.

Reply:
I made it per-Kafka, so you get reminded of the lag when you create a new instance.